modules/node_groups/launchtemplate.tf

```diff
@@ -0,0 +1,94 @@
+data "template_file" "workers_userdata" {
+  for_each = { for k, v in local.node_groups_expanded : k => v if v["create_launch_template"] }
+  template = file("${path.module}/templates/userdata.sh.tpl")
+
+  vars = {
+    bootstrap_extra_args = each.value["bootstrap_extra_args"]
```

Comment:
This variable is not used in the template. I had to remove it in order for a plan to succeed.

Reply:
I remove it